### PR TITLE
feat: internet connectivity checker

### PR DESCRIPTION
**Pull request by Sardar1208**
**- What I did - added internet connectivity checker, updated snackbar wordings**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: lib/view_models/internet_connectivity_checker.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
+
+import 'package:at_wavi_app/view_models/base_model.dart';
+import 'package:internet_connection_checker/internet_connection_checker.dart';
+
+class InternetConnectivityChecker extends BaseModel {
+  bool isInternetAvailable = false;
+  late StreamSubscription _listener;
+
+  checkConnectivity() async {
```
Comment (sachins-geekyants) [inline lib/view_models/internet_connectivity_checker.dart:10]: Where is `checkConnectivity` called?
If it's not getting called, `isInternetAvailable ` will always be false.

Comment (Sardar1208) [reply lib/view_models/internet_connectivity_checker.dart:10]: it is called in home.dart initState() method

Comment (sachins-geekyants) [reply lib/view_models/internet_connectivity_checker.dart:10]: It should be added desktop Home @Sardar1208  ?

Comment (Sardar1208) [reply lib/view_models/internet_connectivity_checker.dart:10]: Yes fixed it.@sachins-geekyants

**Review by sachins-geekyants — DISMISSED** [review]

**Review by sachins-geekyants — APPROVED** [review]

**Pull request merged**